test/controllers/documentation/DocumentationControllerSpec.scala

```diff
@@ -17,7 +17,7 @@ object DocumentationControllerSpec extends PlaySpecification {
       val reverseRouter: ReverseRouter = inject[ReverseRouter]
       val request                      = FakeRequest("GET", reverseRouter.page(None, "2.5.x", "Home"))
       val result                       = await(route(app, request).get)
-      result.header.headers.contains("Link") must beTrue
+      result.header.headers.contains("link") must beTrue
```

Comment:
No sure if this will work, but when checking the headers sent in the browser, the 'link' is lowercase.

It can be that my browser is displaying it lowercase. 

Anyway, I'm not able to run the tests on my own machine nor to debug it. Unclear why it's failing here.

Reply:
Nope, that's not the case. Test still failing.